Watchman.Discord/Areas/Reactions/Contexts/ReactionContext.cs

```diff
@@ -2,7 +2,7 @@
 using Discord;
 using Discord.WebSocket;
 
-namespace Devscord.DiscordFramework
+namespace Watchman.Discord.Areas.Reactions.Contexts
```

Comment:
ta zmiana jest niepotrzebna, lepiej to wrzucić w miejsce gdzie są UserContext, ChannelContext itp.